projects/bp-gallery/src/components/views/exhibitions/ExhibitionOverview.tsx

```diff
@@ -0,0 +1,398 @@
+            {isCurator && (
+              <div className='flex gap-2 flex-row-reverse'>
+                <Button
+                  onClick={e => {
+                    e.stopPropagation();
+                    visit(link);
+                  }}
+                >
+                  {t('exhibition.overview.to-exhibition')}
+                </Button>
+                <Button
+                  onClick={e => {
+                    e.stopPropagation();
+                    visit(editLink);
+                  }}
+                >
+                  {t('common.edit')}
+                </Button>
+              </div>
+            )}
+          </CardContent>
+        </CardActionArea>
+        {isCurator && (
+          <div className='absolute top-2 right-2 bg-white rounded-full'>
+            <IconButton onClick={() => setIsPopupOpen(true)}>
+              <Delete />
+            </IconButton>
+          </div>
+        )}
+      </Card>
+    </div>
+  );
+};
+
+const DeleteModal = ({
+  isPopupOpen,
+  setIsPopupOpen,
+  exhibitionTitle,
+  exhibitionId,
+  refetch,
+}: {
+  isPopupOpen: boolean;
+  setIsPopupOpen: Dispatch<SetStateAction<boolean>>;
+  exhibitionTitle: string;
+  exhibitionId: string;
+  refetch: any;
+}) => {
+  const [deleteExhibition] = useDeleteExhibitionMutation();
+  const deleteThisExhibition = async () => {
+    await deleteExhibition({ variables: { id: exhibitionId } });
+    refetch();
+  };
+  return (
+    <Modal open={isPopupOpen} onClose={() => setIsPopupOpen(false)}>
+      <Box className='absolute w-[50vw] bg-white top-1/2 left-1/2 translate-x-[-50%] translate-y-[-50%] p-8 flex flex-col items-center'>
+        <h3>{t('exhibition.overview.delete-title', { exhibitionName: exhibitionTitle })}</h3>
+        <div className='flex gap-2'>
+          <Button
+            onClick={() => {
+              deleteThisExhibition();
+              setIsPopupOpen(false);
+            }}
+          >
+            {t('common.yes')}
+          </Button>
+          <Button onClick={() => setIsPopupOpen(false)}>{t('common.no')}</Button>
+        </div>
+      </Box>
+    </Modal>
+  );
+};
+
+const ExhibitionOverview = ({
+  archiveId,
+  showTitle,
+  margin,
+  backgroundColor,
+}: {
+  archiveId?: string;
+  showTitle?: boolean;
+  margin?: boolean;
+  backgroundColor?: string;
+}) => {
+  const { data: exhibitionsData, refetch } = useGetExhibitionsQuery({
+    variables: archiveId ? { archiveId } : undefined,
+  });
+  const exhibitions: FlatExhibition[] | undefined =
+    useSimplifiedQueryResponseData(exhibitionsData)?.exhibitions;
+  const { role } = useAuth();
+  const { visit } = useVisit();
+  const { t } = useTranslation();
+  const { isMobile } = useContext(MobileContext);
+  const isCurator = role >= AuthRole.CURATOR;
+  const [showMore, setShowMore] = useState(false);
+  const isOverflow = e => {
+    if (!e) return false;
+    return e.offsetWidth < e.scrollWidth;
+  };
```

Comment:
this does get either a HTMLDivElement or null as value, right? Why is the parameter called e?
```suggestion
  const isOverflow = (node: HTMLDivElement | null) => {
    if (!node) return false;
    return node.offsetWidth < node
```suggestion
  const isOverflow = e => {
    if (!e) return false;
    return e.offsetWidth < e.scrollWidth;
  };
```